src/specific/s_filesystem.c

```diff
@@ -17,3 +27,17 @@ const char *S_File_GetGameDirectory()
     }
     return m_GameDir;
 }
+
+void S_File_CreateDirectory(const char *path)
+{
+    assert(path);
+    int32_t err_check = 0;
+#if defined(_WIN32)
+    err_check = _mkdir(path);
+#else
+    err_check = mkdir(path, 0664);
+#endif
+    if (err_check != 0) {
+        LOG_ERROR("mkdir failed! %s\n", strerror(errno));
```

Comment:
We don't put `\n` in `LOG_*` calls, it does it for us.

Reply:
I'm going to remove this because every screenshot the user takes after the first one will log an "error message" saying the directory already exists.